templates/aws-standalone-cp/templates/apps/nginx-ingress.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: source.toolkit.fluxcd.io/v1beta2
+kind: OCIRepository
+metadata:
+  name: nginx-ingress
+  labels:
+    hmc.mirantis.com/managed: "true"
+spec:
+  interval: 24h
+  url: oci://ghcr.io/nginxinc/charts/nginx-ingress
+  ref:
+    semver: "1.3.2"
+---
+apiVersion: helm.toolkit.fluxcd.io/v2
+kind: HelmRelease
+metadata:
+  name: {{ include "cluster.name" . }}-nginx-ingress
+  labels:
+    hmc.mirantis.com/managed: "true"
+spec:
+  targetNamespace: nginx-ingress
+  storageNamespace: nginx-ingress
+  kubeConfig:
+    secretRef:
+      name: {{ include "cluster.name" . }}-kubeconfig
+  interval: 10m
+  chartRef:
+    kind: OCIRepository
+    name: nginx-ingress
+  install:
+    createNamespace: true
+    remediation:
+      retries: -1
+  values:
```

Comment:
@Schnitzel @wahabmk Do we want to allow overrides for some/all values? I think it also makes sense to allow enabling/disabling of those releases.

Reply:
yea agree we should let the customer overwrite anything they want (especially in this current iteration). In the future we might have tighter managed templates by mirantis that will not allow any changes, but here it's ok.